cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -26,7 +26,7 @@ handle_cluster () {
   local region
   region=$3
   echo "handling cluster: ${cluster_name} in region ${region} with expirationDate: ${expirationDateValue}"
-  ./bin/hiveutil aws-tag-deprovision "${cluster_name}=owned" --region "${region}"
+  timeout -v 10m ./bin/hiveutil aws-tag-deprovision "${cluster_name}=owned" --region "${region}"
```

Comment:
I think you want to alert here so someone can check that cluster and manually reap it.